spec/factories/neighborhood.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :neighborhood do
+    name { Faker::Lorem.word }
```

Comment:
In general, I don't like using `Faker` for factory data, because it's not obvious what the test is looking for. What about "Some neighborhood name #{n}" in conjunction with FactoryBot sequences?